pachyderm/templates/etcd/headless-service.yaml

```diff
@@ -1,4 +1,4 @@
----
+{{- if and (gt .Values.etcd.dynamicNodes 0.0) (eq .Values.pachd.storage.backend "LOCAL" | not) -}}
```

Comment:
what's the reasoning behind the `not local` here?

Reply:
It is not currently present in the local manifest.

Reply:
Sorry for the faff here, I'd actually remove this check (I don't see an issue with creating the service locally (Likely an error with the original manifest) Also I _think_ this enables communication between nodes in a multi node scenario, which we probably want locally